lib/repl.js

```diff
@@ -270,11 +271,7 @@ function REPLServer(prompt,
 
   if (options.terminal && options.useColors === undefined) {
     // If possible, check if stdout supports colors or not.
-    if (options.output.hasColors) {
-      options.useColors = options.output.hasColors();
-    } else if (process.env.NODE_DISABLE_COLORS === undefined) {
-      options.useColors = true;
-    }
+    options.useColors = shouldColorize(options.output) || process.env.NODE_DISABLE_COLORS === undefined;
```

Comment:
I realize this was there before this PR, but it seems weird to me to use `process.env.NODE_DISABLE_COLORS === undefined`, that means that `NODE_DISABLE_COLORS= node script.js` would behave the same as `NODE_DISABLE_COLORS=1 node script.js`, which is weird I think.

Reply:
yeah, can probably be fixed in a follow-up PR

Reply:
@aduh95 tested your example, it will be an empty string, not undefined:

```bash
NODE_DISABLE_COLORS= node -p "util.inspect(process.env.NODE_DISABLE_COLORS)"
''
```